### PR TITLE
Added docs about formsender logs

### PR DESCRIPTION
**Pull request by matthewrsj**

### CODE REVIEW

**Comment by matthewrsj** [comment]
Fixes #44

File: docs/source/errorcodes.rst [hunk 1]
```diff
@@ -1,10 +1,16 @@
 .. _errorcodes:
 
+Error Codes and Logs
+====================
+
+Formsender logs its activity to syslog and tracks errors to send information to
```
Comment (Kennric) [inline docs/source/errorcodes.rst]: This sentence is a hard to parse, maybe a rework to make it clear that there are two ways errors are dealt with?

File: docs/source/errorcodes.rst [hunk 2]
```diff
@@ -48,3 +54,28 @@ These error codes can be handled with a little javascript in your redirect page:
                      "<p style='color:red'>Error number: ", errorNumber, "</p>",
                      "<p style='color:red'>Error message: ", errorMessage, "</p>");
     }
+
+Logs
+----
+
+In addition to the error codes sent to the redirect url, logs are sent to syslog
+on the server or workstation where formsender is running. Information will be
```
Comment (Kennric) [inline docs/source/errorcodes.rst]: I don't know if we need to bother distinguishing server or workstation - just say syslog on the system where formsender is running, that should cover everything.

**Comment by matthewrsj** [comment]
Fixes #45

**Comment by Kennric** [comment]
Looks good. +1
